api/app/config/detekt.yml

```diff
@@ -1,3 +1,6 @@
+build:
+  maxIssues: 3
```

Comment:
Suddenly started getting the below error, every fix I tried did not work, so I introduced a maxIssue count 

`Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end [ImportOrdering]`